doc/tutorials/using_windows_as_uos.rst

```diff
@@ -309,6 +360,21 @@ together with the script used to install Windows 10.
    .. figure:: images/windows_install_6.png
       :align: center
 
+This is the step to install the oracle driver
```

Comment:
Maybe you could have a single line that reads "These are the steps to install the Oracle driver" (instead of repeating it before each picture)?